docs/troubleshooting/25_social-sign-in.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: troubleshooting-social-sign-in
+title: Social Sign-in Troubleshooting
+sidebar_label: Social Sign-in Troubleshooting
+slug: troubleshooting-social-sign-in
+---
+
+### `redirect_uri` mismatch
```

Comment:
1. You should use H2s for headings instead of jumping straight into H3s :)
2. Can you please refrain from using code formatting in headings? Regular `'` will do just fine.

Reply:
Alright, will adjust accordingly